### PR TITLE
initial changes for alert messages

### LINKED ISSUE #32: Login feedback
If you put in a bad login it doesn't do anything. It should say you failed the login.

### PR DESCRIPTION
**Pull request by codedmart**
I removed the validation stuff from this. This should just be the alert message. Accept or change as you see fit. Fixes #32.

### CODE REVIEW

File: web/app/books/support.js [hunk 1]
```diff
@@ -1,10 +1,10 @@
 // @flow
 import React from 'react'
-import _ from 'lodash'
 
 var EMAIL = "serials@orbit.al"
 
 import {makeUpdate} from '../data/update'
+import {Alerts} from '../model/alert'
```
Comment (seanhess) [inline web/app/books/support.js:7]: unused?

**Comment by seanhess** [comment]
Looks good. Thanks!

File: web/app/books/support.js (hunk 1 above)
Comment (codedmart) [inline web/app/books/support.js:7]: Yes good catch. It was before the beta stuff was changed.

File: web/app/model/user.js [hunk 2]
```diff
@@ -33,6 +33,8 @@ export class UserModel {
 
   events: EventEmitter;
 
+  events: EventEmitter;
```
Comment (codedmart) [inline web/app/model/user.js:36]: To that end this looks like a duplicate. I will fix those.

**Pull request merged**